src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todos, Users } from './components/types';
 
 export const App = () => {
+  const [count, setCount] = useState<number>(2);
```

Comment:
No need to store this in State. You can fined the max value of all todos' ids and increment it